### PR TITLE
Use the accepted socket, not the listening one.

### PR DESCRIPTION
**Pull request by mmeeks**
Change-Id: Iecaa90f4c3030b5b2e20bb0e33ef9b75717bb578
Signed-off-by: Michael Meeks <michael.meeks@collabora.com>

* Resolves: # 
* Target version: master 

### TODO

- [ ] ...

### Checklist

- [ ] Code is properly formatted
- [ ] All commits have Change-Id
- [ ] I have run tests with `make check`
- [ ] I have issued `make run` and manually verified that everything looks okay
- [ ] Documentation (manuals or wiki) has been updated or is not required

### CODE REVIEW

File: net/Socket.cpp [hunk 1]
```diff
@@ -743,7 +743,7 @@ std::shared_ptr<Socket> LocalServerSocket::accept()
         // Sanity check this incoming socket
         struct ucred creds;
         socklen_t credSize = sizeof(struct ucred);
-        if (getsockopt(getFD(), SOL_SOCKET, SO_PEERCRED, &creds, &credSize) < 0)
+        if (getsockopt(_socket->getFD(), SOL_SOCKET, SO_PEERCRED, &creds, &credSize) < 0)
         {
             LOG_ERR("Failed to get peer creds on " << getFD() << ' ' << strerror(errno));
```
Comment (kevans91) [inline net/Socket.cpp:748]: `_socket->getFD()` as well?

Comment (kevans91) [reply net/Socket.cpp:748]: I guess the listening fd might actually be more useful here.

**Comment by mmeeks** [comment]
Good point =)

**Review by Ashod — APPROVED** [review]
Looks good, thanks.

**Pull request merged**